### PR TITLE
New package: sowm-1.6

### PR DESCRIPTION
**Pull request by shizonic**

### CODE REVIEW

File: srcpkgs/sowm/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'sowm'
+pkgname=sowm
+version=1.0
+revision=1
+build_style=gnu-makefile
+makedepends="libX11-devel"
+depends="libX11"
```
Comment (Piraty) [inline srcpkgs/sowm/template]: setting this is redundant.
```
<...>
=> sowm-1.0_1: running pre-pkg hook: 04-generate-runtime-deps ...
   SONAME: libX11.so.6 <-> libX11>=1.2_1
<...>
```

File: srcpkgs/sowm/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+short_desc="Itsy bitsy floating window manager "
+maintainer="shizonic <realtiaz@gmail.com>"
+license="MIT"
+homepage="https://github.com/dylanaraps/sowm"
+distfiles="https://github.com/dylanaraps/sowm/archive/${version}.tar.gz"
+checksum=60f3deaf2663fd35da3ac305143350f4b28dd41cea5f4ffc0fd7cf7f1e268098
+
+do_install() {
+	make PREFIX=/usr DESTDIR="${DESTDIR}" install
```
Comment (Piraty) [inline srcpkgs/sowm/template]: it's enough to just vlicense in post_install, as your make cmd matches the default by xbps...

File: srcpkgs/sowm/template [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'sowm'
+pkgname=sowm
+version=1.0
+revision=1
+build_style=gnu-makefile
+makedepends="libX11-devel"
+depends="libX11"
+short_desc="Itsy bitsy floating window manager "
```
Comment (Piraty) [inline srcpkgs/sowm/template]: remove trailing whitespace from string

**Comment by shizonic** [comment]
Sorry for delay. Was on vacation.

**Comment by abenson** [comment]
1.6 is out

**Comment by shizonic** [comment]
Updated.

**Pull request merged**